src/antsibull/cli/__init__.py

```diff
@@ -1 +1,4 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+# SPDX-FileCopyrightText: Ansible Project, 2020
+
```

Comment:
I guess I misunderstood what you mean then. Which identifier do you mean?

Reply:
Some files have

```
# Copyright (c) Ansible Project
# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
# SPDX-License-Identifier: GPL-3.0-or-later
```

while others have

```
# SPDX-License-Identifier: GPL-3.0-or-later
# SPDX-FileCopyrightText: Ansible Project, 2020
```

Reply:
Ok, I think I've now normalized them. Note that sometimes `COPYING` is used instead of `LICENSES/GPL-3.0-or-later.txt`, namely in the notices that will end up in the ansible tarball (there the license file is called `COPYING`).